android/app/src/main/java/com/mattermost/helpers/PushNotificationDataHelper.kt

```diff
@@ -38,12 +41,20 @@ class PushNotificationDataRunnable {
                     var userIdsToLoad: ReadableArray? = null
                     var usernamesToLoad: ReadableArray? = null
 
+                    var threads: ReadableArray? = null
+                    var usersFromThreads: ReadableArray? = null
+                    var receivingThreads = false
```

Comment:
```suggestion
                    var receivingThreads = isCRTEnabled && rootId != ""
```

you may need to check if its null or empty